src/pybind/mgr/ssh/module.py

```diff
@@ -968,11 +971,12 @@ def create_osds(self, drive_group):
         return self.get_hosts().then(lambda hosts: self._create_osd(hosts, drive_group))
 
     def remove_osds(self, name):
```

Comment:
The overridden method in the base class has different parameters:
https://github.com/ceph/ceph/blob/b12107a5532a7fbba44046d3b4768ff66e675477/src/pybind/mgr/orchestrator.py#L868-L873